common.py

```diff
@@ -1133,7 +1140,7 @@ def cutbased_ddt(cols, lumi, ddt_map_file, xrootd_url):
     ecfm2b1 = cols.to_numpy(['ecfm2b1']).ravel()
     weight = get_event_weight(cols, lumi)
 
-    ddt_val = calculate_varDDT(mT, pT, rho, ecfm2b1, weight, 0.09, ddt_map_file)
+    ddt_val = calculate_varDDT(mT, pT, rho, ecfm2b1, weight, 0.12, ddt_map_file)
```

Comment:
Is this 0.12 the new optimal cut value for ECF? Maybe we need to elevate this to be a parameter we pass into the function.